### DISCUSSION TITLE
What's New window

### DISCUSSION THREAD

**Discussion by smk78** [SasView/sasview#2645] [Ideas]
**Describe the bug**
Not a bug, per se, but an observation which, now I've been confronted by it multiple times at Code Camp, I'm beginning to think is the wrong way round.

The new What's New dialog is usefully extensible in that you can have it provide 'What _Was_ New' last release, or the release before that, etc. But if you take advantage of this functionality, it is presented _oldest_ first.

What I'm now wondering is whether it would be better presented newest first; ie, so that What's New for the release for the version of SasView you have just run is what you see first, and the information for earlier versions (if provided) is what you get when you click 'Next'?

**Comment by lucas-wilkins** [comment]
Yeah, maybe, I guess it depends how far back the contents goes. If it is too far, it will give pointless stuff... then again, maybe need to carefully curate what's in the whats new box.

**Comment by butlerpd** [comment]
I agree that whatever the case, if there are multiple versions being listed, the order should be newest first not oldest first. That is how our release notes work and most CVs list recent papers first (what have you done recently 😄). There are a few cases where I think I've seen the reverse order but generally most recent first seems appropriate.

That said, Not sure the "what's new" box should have the whole history? the release notes are available from the GUI already right? Just some thoughts

**Reply by lucas-wilkins** [reply]
What's new is for what we want to make users to be aware of, just need to be pragmatic about what goes in.

I wouldn't necessarily put the release notes in it. More, a summary of the big changes and anything that might catch people out.

**Comment by krzywon** [comment]
This exists in v6.0.0. Closing, but feel free to reopen
